botx/clients/methods/v3/stickers/sticker_pack_list.py

```diff
@@ -0,0 +1,41 @@
+"""Method for getting sticker pack list."""
+from typing import Optional
+from uuid import UUID
+
+from botx.clients.methods.base import AuthorizedBotXMethod
+from botx.clients.types.http import HTTPRequest
+from botx.models.stickers import StickerPackList
+
+
+class GetStickerPackList(AuthorizedBotXMethod[StickerPackList]):
+    """Method for getting sticker pack list."""
+
+    __url__ = "/api/v3/botx/stickers/packs"
+    __method__ = "GET"
+    __returning__ = StickerPackList
+
+    #: Author ID.
+    user_huid: Optional[UUID]
+
+    #: Count of returning value.
+    limit: int = 1
```

Comment:
You have already defined `1` as default in `StickersMixin.get_sticker_pack_list`